Geometry/HcalCommonData/data/v1/hcalsenspmf.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
```

Comment:
Full path is wrong. It should be `Geometry/HcalCommonData/data/hcalsenspmf/v1/hcalsenspmf.xml`.

Reply:
The previous path was Geometry/HcalCommonData/data/hcalsenspmf.xml

Reply:
Yes, the new directory `hcalsenspmf` has to be created.